holodeck/environments.py

```diff
@@ -253,6 +245,37 @@ def write_to_command_buffer(self, to_write):
         for index, val in enumerate(input_bytes):
             self._command_buffer_ptr[index] = val
 
+    def set_hyper_parameter(self, agent_name, parameter_index, value):
+        """Sets a specific hyper parameter on a specific agent.
+
+        Positional Arguments:
+        agent_name -- the name of the agent
+        parameter_index -- The index of the parameter to set
+        value -- The value to set the parameter to.
+        """
+        if agent_name not in self._hyper_parameters_map:
+            print("Agent does not exist: ", agent_name)
```

Comment:
I think it would be better to raise a HolodeckError here, it more forcefully tells the user that what they're trying to do is wrong. What do you think?